pages/search.js

```diff
@@ -203,6 +202,31 @@ export default function Search({
     );
   };
 
+  const handleClearFilter = () => {
+    const params = new URLSearchParams({ query: searchTerm });
+    params.forEach((e) => {
```

Comment:
I find _e_ a bit ambiguous, consider renaming it to a more descriptive and meaningful name. This can enhance code readability and understanding. Choose a name that reflects the purpose or content of the variable to make the code more self-explanatory.